hack/ukify/main.go

```diff
@@ -47,18 +49,37 @@ var (
 	output         string
 )
 
-func sbSign(input string) (string, error) {
+func sign(input string) (string, error) {
 	out := input + ".signed"
 
 	if err := os.RemoveAll(out); err != nil {
 		return "", err
 	}
 
-	cmd := exec.Command("sbsign", "--key", signingKey, "--cert", signingCert, "--output", out, input)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	pem, err := x509.NewCertificateAndKeyFromFiles(signingCert, signingKey)
+	if err != nil {
+		return "", err
+	}
+	cert, err := pem.GetCert()
+	if err != nil {
+		return "", err
+	}
+	key, err := pem.GetRSAKey()
+	if err != nil {
+		return "", err
+	}
```

Comment:
extract into function?

Reply:
just within the ukify `main.go` or into a module elsewhere?

Reply:
i think under certs is fine

Reply:
I didn't extract since we don't import `gen-certs` into `ukify` for now.